### ISSUE TITLE
[SUGGESTION]: Add a second type filter for held items

### ISSUE THREAD

**Issue by Zehirozzy** [pokeclicker/pokeclicker#5096]
### Current Behavior

When giving held items, you can only filter on one type.

### Improved Behavior

Being able to filter on a second type when giving held items.

### Reason for change

Being able to easily prioritize mono types when giving held items.

### Detailed Description

If I want to prioritize giving held items to mono types Pokémon, I must look in the Pokédex, or the wiki, or even know every type from memory.
I'd like to have 2 type filters, to display only mono types Pokémon.

**Comment by Razorflamekun** [comment]
"If I want to prioritize giving held items to mono types Pokémon, I must look in the Pokédex, or the wiki, or even know every type from memory."  Actually, this is not really true because if your selected mono-type is, for example, Steel, you can literally just select the Steel type and literally every single Pokemon of which **either one of the two types match** should return the result you are looking for.

**Comment by Zehirozzy** [comment]
Yes, but dual types will be mixed up with mono types then. What I would like to have is an option to only display mono types, with the help of two filters, like in the Pokédex.
Or is that already possible with held items ?

**Comment by CypherX** [comment]
Resolved by #5684
